### PR TITLE
Proposals/demising above grade wall area

### PR DESCRIPTION
**Pull request by JieXiong9119**
See proposal
Implementation in schema added

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="DemisingAboveGradeWallArea" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Above grade demising wall area. (ft2)</xs:documentation>
```
Comment (macintoshpie) [inline BuildingSync.xsd:801]: We might want to reverence the ASHRAE standard here. If you search in the schema there are a few places where we do this

Comment (JieXiong9119) [reply BuildingSync.xsd:801]: > We might want to reverence the ASHRAE standard here. If you search in the schema there are a few places where we do this

Could you point me to those places?

File: BuildingSync.xsd [hunk 2]
```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="DemisingAboveGradeWallArea" minOccurs="0">
```
Comment (corymosiman12) [inline BuildingSync.xsd:799]: I propose changing the element name to: `TotalCommonConditionedAboveGradeWallArea` and the documentation to: `The total area of the above grade wall common with other conditioned buildings`

File: proposals/2021/Add DemisingAboveGradeWallArea.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        </xs:complexType>
+      </xs:element>
+    ...
+      <xs:element name="DemisingAboveGradeWallArea" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Above grade demising wall area. (ft2)</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:decimal">
```
Comment (corymosiman12) [inline proposals/2021/Add DemisingAboveGradeWallArea.md:54]: Jie let's also add this as a non negative decimal.

File: proposals/2021/Add DemisingAboveGradeWallArea.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+              <xs:attribute ref="auc:Source"/>
+            </xs:extension>
+          </xs:simpleContent>
+        </xs:complexType>
+      </xs:element>
+    ...
+```
+
+## References
+https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-25130.pdf
```
Comment (corymosiman12) [inline proposals/2021/Add DemisingAboveGradeWallArea.md:64]: For reference, you can use:
- https://help.buildingenergyscore.com/support/solutions/articles/8000053362-facility-description-construction (even though their usage of the term demising wall is incorrect)
- ASHRAE Standard 211 6.2.1.2.b

**Review by corymosiman12 — CHANGES_REQUESTED** [review]
Hi Jie - minor requested changes - thanks for putting together!

File: BuildingSync.xsd (hunk 1 above)
Comment (macintoshpie) [reply BuildingSync.xsd:801]: Here's an example: https://github.com/BuildingSync/schema/blob/458dcc218d771d2520cd27aa7a87b63fac560e14/BuildingSync.xsd#L4106

**Review by yan130 — APPROVED** [review]

**Review by corymosiman12 — APPROVED** [review]

File: proposals/2021/Add DemisingAboveGradeWallArea.md (hunk 3 above)
Comment (nllong) [reply proposals/2021/Add DemisingAboveGradeWallArea.md]: excellent!

File: proposals/2021/Add DemisingAboveGradeWallArea.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+```xml
+
+    <Facilities>
+        <Facility ID="F1">
+            <Sites>
+                <Site>
+                    <Buildings>
+                        <Building ID="G1">
+                            <DemisingAboveGradeWallArea>1000</DemisingAboveGradeWallArea>
```
[the remark this answers is not in the document]
Comment (nllong) [reply proposals/2021/Add DemisingAboveGradeWallArea.md]: The value is set to nonNegativeDecimal. So this is just a visual thing.

**Pull request merged**